modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1935,6 +1935,18 @@
         defaults = "" + DFLT_DUMP_TX_COLLISIONS_INTERVAL)
     public static final String IGNITE_DUMP_TX_COLLISIONS_INTERVAL = "IGNITE_DUMP_TX_COLLISIONS_INTERVAL";
 
+    /**
+     * Set to true only during the junit tests.
```

Comment:
I'd also add extra line similar to : "Signals that the cluster is running in a test environment."